packages/live-share-media/src/VolumeLimiter.ts

```diff
@@ -83,100 +84,85 @@ export class VolumeLimiter {
     /**
      * Amount of time, in seconds, it should take to ramp the volume down to the desired level.
      */
-    public get rampDownDuration(): number {
-        return this._rampDownDuration.seconds;
+    public get rampDuration(): number {
+        return this._rampDuration.seconds;
     }
 
-    public set rampDownDuration(value: number) {
-        this._rampDownDuration.seconds = Math.abs(value);
+    public set rampDuration(value: number) {
+        this._rampDuration.seconds = Math.abs(value);
     }
 
     /**
-     * Amount of time, in seconds, it should take to ramp the volume back up to the original level.
+     * Limits volume based on `level` and `levelType` properties.
      */
-     public get rampUpDuration(): number {
-        return this._rampUpDuration.seconds;
-    }
-
-    public set rampUpDuration(value: number) {
-        this._rampUpDuration.seconds = Math.abs(value);
+    public enableLimit(): void {
+        this._limited = true;
+        this.startAdjusting();
     }
 
     /**
-     * Temporarily lowers an audio sources volume.
+     * disables volume limit.
```

Comment:
Might rename to `start` and `stop`. It is already named `VolumeLimiter` so I'm not sure if it needs to be anything too redundant. Also would make sure comments are capitalized. This should also be aligned with `enableLimit` in that it references that it will ramp up to the specified max volume